hopr/hopr-lib/Cargo.toml

```diff
@@ -45,6 +45,7 @@ async-std = { workspace = true, optional = true }
 atomic_enum = { workspace = true }
 const_format = "0.2.32"
 futures = { workspace = true }
+futures-concurrency = { workspace = true }
```

Comment:
Version bump of crate missing.

Reply:
The hopr-lib version is the release version, it is bumped automatically on release.